src/style.css

```
@@ -0,0 +1,139 @@
+* {
+
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.form {
+
+  position: relative;
+  margin-top: 20px;
+  margin-right: auto;
+  margin-bottom: 0;
+  margin-left: auto;
+}
+
+.label {
+
+  position: relative;
+  display: block;
+  width: 100%;
+
+}
+
+.input_large {
+
+  outline: none;
+  padding: 0 0 0 62px;
```

kind of fine, yet `padding-left` would be more explicit
```suggestion
  padding-left: 62px;
```